### ISSUE TITLE
NTR: Glutathione reductase low in erythrocytes (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8237]
New term request Glutathione reductase low in erythrocytes (elthat_200327165323): RBCs, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000525
- label: Reduced erythrocyte glutathione reductase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:17185460
- def: Activity of glutathione reductase (EC 1.6.4.2) in erythrocytes below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8237
- synonyms:
